src/main/flight/pid.c

```diff
@@ -445,19 +446,60 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #ifdef USE_GPS_RESCUE
     angle += gpsRescueAngle[axis] / 100; // ANGLE IS IN CENTIDEGREES
 #endif
-    angle = constrainf(angle, -levelAngleLimit, levelAngleLimit);
     const float errorAngle = angle - ((attitude.raw[axis] - angleTrim->raw[axis]) / 10.0f);
+    const float levelPGainBoost = (1.0f + (fabsf(errorAngle) / levelAngleLimit));
+    float angleFeedforward = 0;
+    //angle loop feedforward
+#ifdef USE_FEEDFORWARD
+    if (newRcFrame){
+        const float feedforwardJitterFactor = pidGetFeedforwardJitterFactor();
+        const float rcCommandDelta = fabsf(getRcCommandDelta(axis));
+        float angleFeedforwardInput = pidRuntime.angleFeedforward[axis];
+        uint8_t duplicateCount = getFeedforwardDuplicateCount(axis);
+        if (rcCommandDelta) {
+            const float rxRateHz = 1e6f / getCurrentRxRefreshRate();
+            angleFeedforwardInput = (angle - pidRuntime.previousAngle[axis]) * rxRateHz;
+            //jitter attenuation
+            float jitterAttenuator = 1.0f;
```

Comment:
The jitter reduction code is really important for angle feedforward since there is no expo to attenuate it.
I've tested it, and it is fantastic.

Maybe we could refactor this to get the `jitterAttenuator` calculated once in rc.c and brought both to pid.c and feedforward.c via a `getKitterAttenuator()` function, so that we don't do the maths twice?

That would be awesome.